backend/databases/qdrant.py

```
@@ -103,3 +103,15 @@ def search_similar_items(query_vector: list, limit: int = 10):
         # MAINTAINABILITY ISSUE - Generic exception handling
         print(e)
         return []
+
+def delete_item_vector(item_id: int):
+    """Delete item embedding from Qdrant"""
+    try:
+        qdrant_client.delete(
+            collection_name=QDRANT_COLLECTION_NAME,
+            points_selector=PointIdsList(points=[item_id])
+        )
+        return True
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)